app/components/Calendar.tsx

```diff
@@ -10,33 +14,65 @@ const initialEventsList: CalEvent[] = [
     title: 'timed event',
     start: moment().add(1, 'hour').toDate(),
     end: moment().add(2, 'hours').toDate(),
+    description: '',
+    location: '',
```

Comment:
Can we move this data to constants?

Reply:
Yes, we will be removing these from here entirely once the global store is set. Coming in next PRs.